### PR TITLE
Updated index.md file.

### PR DESCRIPTION
**Pull request by moinshaikh**
Create introductory article about HTML Tr tag with definition and code example.

Replace this sentence with a brief description of the awesome changes your PR has. 😊

---

## ✅️ By submitting this PR, I have verified the following

- [x] Added descriptive name to PR
  - Your PR should NOT be called `Update index.md` since it does not help the maintainer understand what has been changed.
  - For example, if you create a **Variables** article inside the **Python** directory, the pull request title should be **Python: add Variables article**.
  - Other examples are **Git: edit Git Commit article** or **PHP: create PHP section and add Data Structures article**
- [x] Reviewed necessary formatting guidelines in [`CONTRIBUTING.md`](https://github.com/freeCodeCamp/guides/blob/master/CONTRIBUTING.md).
- [x] No plagiarized, duplicate or repetitive  content that has been directly copied from another source.

### CODE REVIEW

**Comment by moinshaikh** [comment]
This PR is regarding a newly created Guide article on HTML Tr tag topic. I have added introductory information and example of the HTML Tr tag.

File: src/pages/html/elements/tr-tag/index.md [hunk 1]
```diff
@@ -1,15 +1,34 @@
 ---
 title: Tr Tag
 ---
-## Tr Tag
+<p>The <code class="language-text">&lt;tr&gt;</code> tag defines a standard row in an HTML table on a web page.</p>
+<p>The <code class="language-text">&lt;tr&gt;</code> tag houses the <code class="language-text">&lt;th&gt;</code> and <code class="language-text">&lt;td&gt;</code> tags, also known as Table Heaeder and Table Data respectively.</p>
```
Comment (RiccardoLandolfo) [inline src/pages/html/elements/tr-tag/index.md]: Typo on line 5: Heaeder

Comment (moinshaikh) [reply src/pages/html/elements/tr-tag/index.md]: Thanks for noticing. Fixed it.

Comment (RiccardoLandolfo) [reply src/pages/html/elements/tr-tag/index.md]: Np, you should also change the title as mentioned in the guide.

**Comment by paulywill** [comment]
@moinshaikh Awesome work!
Dwight thanks

**Pull request merged**